qiskit/pulse/configuration.py

```diff
@@ -21,6 +21,46 @@
 from .exceptions import PulseError
 
 
+class Kernel:
+    """Settings for this Kernel, which is responsible for integrating time series (raw) data
+    into IQ points.
+    """
+
+    def __init__(self, name: Optional[str] = None, **params):
+        """Create new kernel.
+
+        Args:
+            name: Name of kernel to be used
+            params: Any settings for kerneling.
+        """
+        self.name = name
+        self.params = params['params']
+
+    def __repr__(self):
+        return "{}({}{})".format(self.__class__.__name__,
+                                 "'" + self.name + "', " or "",
+                                 self.params)
```

Comment:
like `params: Optional[Dict] = None`? That might help some of the issues I've been seeing, since the tests have been doing stuff like `Kernel('name', params={'param':value})`. This in turn yields `params={'params':{'param':value}}` and it's been confusing debugging

Reply:
yes, I think that might be simpler?